ChainSafe/web3.unity
# Web3auth bug fixes

Block explorer url population fixed.
Custom token display fixed.
Auto tx issue fixed.

To Test:
Login with the web3auth scene with auto transactions enabled.

Expected Results:
Change server settings to see the block explorer populate on RPC selection.
Test auto tx calls in the sample scene and check they are working.
Add a custom token and check it's displaying correctly.

closes #1032 & #1033
